README.md

```diff
@@ -103,7 +103,7 @@ An array of literal objects in the form of:
 
 ```js
 import selectOne  from 'co-postgres-queries/queries/selectOne';
-selectOne({ table, idFields, returnFields })(db)(entity)
+selectOne({ table, idFields, returnFields })(entity)
```

Comment:
Shouldn't `idFields` be renamed as `primaryKey` and accept either an array or a value ?